ovn-tester/ovn_tester.py

```diff
@@ -221,28 +272,49 @@ def set_ssl_keys(cluster_cfg):
     Stream.ssl_set_ca_cert_file(cluster_cfg.ssl_cacert)
 
 
-def prepare_test(central_node, worker_nodes, cluster_cfg, brex_cfg):
+def prepare_test(central_nodes, worker_nodes, cluster_cfg, brex_cfg):
     if cluster_cfg.enable_ssl:
         set_ssl_keys(cluster_cfg)
-    ovn = Cluster(central_node, worker_nodes, cluster_cfg, brex_cfg)
-    with Context(ovn, "prepare_test"):
-        ovn.start()
-    return ovn
-
-
-def run_base_cluster_bringup(ovn, bringup_cfg, global_cfg):
-    # create ovn topology
-    with Context(ovn, "base_cluster_bringup", len(ovn.worker_nodes)) as ctx:
-        ovn.create_cluster_router("lr-cluster")
-        ovn.create_cluster_join_switch("ls-join")
-        ovn.create_cluster_load_balancer("lb-cluster", global_cfg)
+
+    clusters = [
+        Cluster(central_nodes[i], worker_nodes[i], cluster_cfg, brex_cfg)
```

Comment:
This always just creates a cluster with a single central node and a single worker.